install/kubernetes/helm/istio/charts/security/templates/deployment.yaml

```diff
@@ -60,12 +60,29 @@ spec:
           {{- if .Values.global.trustDomain }}
             - --trust-domain={{ .Values.global.trustDomain }}
           {{- end }}
+          {{- if .Values.healthChecking }}
+            - --liveness-probe-path=/tmp/ca.liveness # path to the liveness health check status file
+            - --liveness-probe-interval=60s # interval for health check file update
+            - --probe-check-interval=15s    # interval for health status check
+          {{- end }}
```

Comment:
why the {{ end }}with the same if condition on line 68?  Couldn't this all be one block instead?

Reply:
I think this is more clear how it is actually since they are logically separate areas

Reply:
leave a todo: remove http/version one post 1.2 on master?

Reply:
I removed it. I don't think there's a risk removing the httpGet health check. It's not doing meaningful work as I see, and confuses people to think it's doing actual health check.

Reply:
i'll go with consensus.